examples/getreadme/main.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
```

Comment:
close. actually in `github/repos_contents_test.go`, right after the existing `TestRepositoriesService_GetReadme` test.  So we would just add a `ExampleRepositoriesService_GetReadme` there